fe/fe-core/src/main/java/com/starrocks/fs/hdfs/HdfsFsManager.java

```diff
@@ -1280,7 +1280,7 @@ public void renamePath(String srcPath, String destPath, Map<String, String> load
                 throw new UserException("failed to rename path from " + srcPath + " to " + destPath);
             }
         } catch (IOException e) {
-            LOG.error("errors while rename path from " + srcPath + " to " + destPath);
+            LOG.error("errors while rename path from " + srcPath + " to " + destPath, e);
```

Comment:
Because this error is job-level, the level of warning is enough.
Often error is used for system-level issues.

Reply:
Didn't intend to change log level here, just wanna make sure exception printed to logs